ipm/2.4.0.md

```diff
@@ -0,0 +1,36 @@
+# Eaton Intelligent Power Manager Editions 2.4.0 - Release note 
+
+*December 2021*
+
+## Scope
```

Comment:
: missing at the end of § (or remove previous in 2.3.1)

Reply:
applied the : to 2.4.0 too